static/js/services/markdown.js

```diff
@@ -0,0 +1,60 @@
+var inboxServices = angular.module('inboxServices');
```

Comment:
@mandric where is this one used?  If it's already available in the webapp then of course we should be using that.

@garethbowen this is not our own implementation, it's from enketo.  They removed it from their core but we were relying on it for parts of our forms.  It is moved to a service from the enketo notes widget.